6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<ul><li>"Now, we taka a hierarchical model." Should probably be something like "Now we will construct a hierarchical model."</li></ul>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Replaced "Now, we taka a hierarchical model." with "Now, we will construct a hierarchical model."